### PR TITLE
ON-39807 # update `searchCivicPlusHCMSContentItems` to call HCMS dire…

### PR DESCRIPTION
**Pull request by divporter**
…ctly

### CODE REVIEW

File: src/services/integration-elements.ts [hunk 1]
```diff
@@ -961,6 +962,38 @@ export async function getAPINSWLiquorLicence(
   }
 }
 
+const CP_HCMS_TOKEN_KEY = 'CP_HCMS_TOKEN'
+function getCPHCMSTokenFromStorage() {
+  const localStorageValue = localStorage.getItem(CP_HCMS_TOKEN_KEY)
```
Comment (mymattcarroll) [inline src/services/integration-elements.ts:967]: Need to join the `formId` to this to allow different menu items with different content.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - `formService.getGeoscapeReverseGeocoding()`
+- **BREAKING** update `searchCivicPlusHCMSContentItems` to call HCMS directly
```
Comment (mymattcarroll) [inline CHANGELOG.md:13]: This should not be a breaking change and should prob be under the `### Changed` heading.

File: src/services/integration-elements.ts [hunk 3]
```diff
@@ -1030,17 +1066,30 @@ export async function searchCivicPlusHCMSContentItems({
   abortSignal?: AbortSignal
 }): Promise<CivicPlusHCMSContentItemsResult> {
   try {
-    const url = new URL(
-      `${tenants.current.apiOrigin}/forms-apps/${formsAppId}/forms/${formId}/cp-hcms-content-items`,
-    )
+    const {
+      auth: access_token,
```
Comment (mymattcarroll) [inline src/services/integration-elements.ts:1070]:
```suggestion
      auth: { access_token },
```

File: CHANGELOG.md (hunk 2 above)
Comment (divporter) [reply CHANGELOG.md:13]: well, previous versions of apps won't work unless we bring back the endpoint in the api.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- **BREAKING** update `searchCivicPlusHCMSContentItems` to call HCMS directly
```
Comment (mymattcarroll) [inline CHANGELOG.md:12]: This is not a breaking change to this package. Breaking (i.e. major semver bump) means that updating to the new package requires code changes.

File: CHANGELOG.md (hunk 2 above)
Comment (mymattcarroll) [reply CHANGELOG.md]: The api changes do not dictate the semver version of this package.

**Pull request merged**